hub-templates/basehub/values.yaml

```diff
@@ -106,7 +106,7 @@ jupyterhub:
       hub.jupyter.org/node-purpose: user
     image:
       name: set_automatically_by_automation
-      tag: e3bbe25
+      tag: "135823b"
```

Comment:
Probably a stupid question, but curious if the `"`around the image tag make any difference here?

Reply:
https://github.com/helm/helm/issues/1694 happens whenever a hash is either just a number, or a string of the form `123e44` (it is taken as a number in scientific notation). So I started putting quotes around tags as a precaution...

Reply:
Interesting. Thanks for explaining this!